nxontology_ml/model.py

```diff
@@ -0,0 +1,152 @@
+from collections.abc import Iterable
+from dataclasses import dataclass
+from enum import Enum
+from itertools import islice
+from pathlib import Path
+from typing import Any
+
+import numpy as np
+import pandas as pd
+from catboost import CatBoostClassifier, FeaturesData, Pool
+from sklearn.metrics import classification_report
+
+from nxontology_ml.efo import get_efo_otar_slim
+from nxontology_ml.utils import ROOT_DIR
+
+
+@dataclass
+class TrainingRecord:
+    efo_otar_slim_id: str
+    efo_label: str
+    rs_classification: str
+
+
+class FeatureType(Enum):
```

Comment:
Nice, we often use `StrEnum` / sublcassing `(str, Enum)` for things like this such that the values remain readable. Not a strong preference and if integers are needed then ignore this.

Reply:
Good input, will update!